CHANGELOG/CHANGELOG-0.11.md

```diff
@@ -58,7 +61,10 @@ Changes since `v0.11.1`:
 
 ## v0.11.1
 
-Changes since `0.11.1`:
+IMPORTANT: Avoid using this release due to the corrupted [Topology CRD specification](https://github.com/kubernetes-sigs/kueue/issues/4850).
+When upgrading to the newer version please reinstall Kueue. If you are not using TopologyAwareScheduling the upgrading is not urgent.
+
+Changes since `v0.11.0`:
```

Comment:
Here was the conflict because on 0.11 branch we have `Changes since `0.11.1`:. I fixed it to `v0.11.0`